pkg/chrootarchive/archive_windows.go

```diff
@@ -12,7 +12,7 @@ func chroot(path string) error {
 	return nil
 }
 
-func invokeUnpack(decompressedArchive io.ReadCloser,
+func invokeUnpack(decompressedArchive io.Reader,
```

Comment:
ok, that explains~ ; while all other invokeUnpack func are with io.Reader and pass to archive.Unpack only need it to be an io.Reader, even r could be io.ReadCloser that can still work.

I think you can change this line in a separate commit and won't break anything even before merge this.